bids/layout/layout.py

```diff
@@ -533,7 +533,7 @@ def to_df(self, metadata=False, **filters):
             data.loc[o] = pd.Series(dtype=float)
 
         return data.reset_index()
-
+    
```

Comment:
Let's clean this up.

```suggestion

```

Reply:
Hmm. Not sure why it interpreted removing the spaces as removing the line. Dammit github.